src/Microsoft.Deployment.DotNet.Releases/src/Product.cs

```diff
@@ -175,7 +175,7 @@ public bool IsOutOfSupport()
         /// <param name="address">The URL pointing to the releases.json file to use.</param>
         /// <param name="product">The <see cref="Product"/> to link to the releases.</param>
         /// <returns></returns>
-        public static async Task<ReadOnlyCollection<ProductRelease>> GetReleasesAsync(Uri address, Product product)
+        public async Task<ReadOnlyCollection<ProductRelease>> GetReleasesAsync(Uri address, Product product)
```

Comment:
I was expecting the `product` parameter would be removed with the change to make this an instance method.  

Consider grouping with the other GetReleasesAsync overloads.

Reply:
You are correct. I've fixed this in the last commit.